static/js/translator/page.tsx

```diff
@@ -140,10 +132,6 @@ export class Page extends React.Component<PagePropType, PageStateType> {
           bindings={this.state.bindings}
           constraints={this.state.constraints}
         ></Translation>
-        <a href="https://console.cloud.google.com/bigquery?sq=443333369001:c5d2972c96704d65a3091ee15d6f0c87">
```

Comment:
Are we removing BQ link in general?  That seems useful?

Reply:
This link can not update with the actual query which is misleading here. I want to update the translator here to use a real table name so we can copy over the SQL into GCP console.